config/v3_4_experimental/types/config.go

```diff
@@ -25,3 +31,50 @@ var (
 		PreRelease: "experimental",
 	}
 )
+
+func (cfg Config) Validate(c path.ContextPath) (r report.Report) {
+	systemdPath := "/etc/systemd/system/"
+	units := make(map[string]string)
+
+	for _, unit := range cfg.Systemd.Units {
+		if unit.Dropins == nil {
+			units[unit.Name] = systemdPath + unit.Name
```

Comment:
Whether or not we create the unit file depends on whether its `contents` field is specified, not on whether it has dropins.  (And in any event, slices can be non-nil and empty.)

Reply:
That's good to know. I'll update this condition. Can the `Contents` field be specified AND `Dropins` being specified as well? Or is it only one or the other?

Reply:
To answer your question: either, both, or neither can be specified.